src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiNewChat.java

```diff
@@ -217,4 +219,24 @@ private void getChatComponent(int p_getChatComponent_1_, int p_getChatComponent_
             callbackInfo.setReturnValue(null);
         }
     }
+
+    @Redirect(method = "setChatLine", at = @At(value = "INVOKE", target = "Ljava/util/List;size()I", ordinal = 0))
+    private int hookNoLengthLimit(List<ChatLine> list) {
+        ChatControl chatControl = ChatControl.INSTANCE;
```

Comment:
```suggestion
        final ChatControl chatControl = ChatControl.INSTANCE;
```

`chatControl` in `hookChatClear` is declared as final, why not be consistent?